src/services/pcn-iptables/src/datapaths/Iptables_ChainSelector_dp.c

```diff
@@ -132,6 +132,8 @@ static int handle_rx(struct CTXTYPE *ctx, struct pkt_metadata *md) {
   #if _INGRESS_ALLOWLOGIC
       pcn_log(ctx, LOG_DEBUG, "INGRESS LOGIC PASS. No rules for INPUT and FORWARD, and default is ACCEPT. ");
       updateForwardingDecision(PASS_LABELING);
+      incrementDefaultCountersInput(md->packet_len);
```

Comment:
Update `INPUT` counters is not always correct since some pkts could belong to `FORWARD` chain.

Reply:
Then I think just mention it in release note withtout fixing it, in this scenario counting and optimization seems to exclude each other. or we can add a option to allow user to pick up their choice.